src/bootstrap/src/core/config/config.rs

```diff
@@ -1226,6 +1226,13 @@ impl Config {
         self.out.join(self.host_target).join("ci-rustc")
     }
 
+    pub(crate) fn libgccjit_root(&self) -> PathBuf {
```

Comment:
Considering we don't store GCC in the same location depending whether we build it or we download it, I found it better to handle it in one place instead, hence the creation of this method.